src/components/ActorDetails.vue

```diff
@@ -0,0 +1,325 @@
+          </div>
+          <div class="wrapper">
+            <div class="pictures" v-for="image in images" :key="image.id">
+              <router-link :to="'/' + image.media_type + '/' + image.id"
+                ><img
+                  :src="
+                    'https://themoviedb.org/t/p/w130_and_h195_face' +
+                    image.poster_path
+                  "
+              /></router-link>
+              <router-link
+                class="sub-link"
+                :to="'/' + image.media_type + '/' + image.id"
+                ><i>{{ image.original_title }}</i></router-link
+              >
+              <router-link
+                class="sub-link"
+                :to="'/' + image.media_type + '/' + image.id"
+                ><i>{{ image.name }}</i></router-link
+              >
+            </div>
+          </div>
+
+          <div class="cast">
+            <h3 v-if="casts && casts.length > 0">Acting</h3>
+            <div class="acting" v-if="casts && casts.length > 0">
+              <div
+                v-for="cast in casts"
+                :key="cast.credit_id"
+                class="in_acting"
+              >
+                <i v-if="cast.release_date"
+                  >{{ cast.release_date.slice(0, 4) }}
+                </i>
+                <i v-else> —— </i>
+                <i>◯ </i>
+                <router-link
+                  class="sub-link"
+                  :to="'/' + cast.media_type + '/' + cast.id"
+                  ><strong v-if="cast.original_title"
+                    >{{ cast.original_title }}
+                  </strong></router-link
+                >
+                <router-link
+                  class="sub-link"
+                  :to="'/' + cast.media_type + '/' + cast.id"
+                  ><strong v-if="cast.original_name">{{
+                    cast.original_name
+                  }}</strong></router-link
+                >
+                <small v-if="cast.character"> as {{ cast.character }}</small>
+              </div>
+            </div>
+            <h3 v-if="departmentProduction && departmentProduction.length">
+              Production
+            </h3>
+            <Department :endpointType="departmentProduction"></Department>
+            <h3 v-if="departmentCrew && departmentCrew.length">Crew</h3>
+            <Department :endpointType="departmentCrew"></Department>
+            <h3 v-if="departmentDirecting && departmentDirecting.length">
+              Directing
+            </h3>
+            <Department :endpointType="departmentDirecting"></Department>
+            <h3 v-if="departmentWriting && departmentWriting.length">
+              Writing
+            </h3>
+            <Department :endpointType="departmentWriting"></Department>
+            <h3 v-if="departmentCreator && departmentCreator.length">
+              Creator
+            </h3>
+            <Department :endpointType="departmentCreator"></Department>
```

Comment:
Rename argument from `endpointType` to `departments`. `endpointType` does not make any sense.